tests/gentropy/dataset/test_study_locus.py

```diff
@@ -151,41 +151,6 @@ def test_find_overlaps(mock_study_locus: StudyLocus) -> None:
     assert isinstance(mock_study_locus.find_overlaps(), StudyLocusOverlap)
```

Comment:
Could be useful to see some example test (perhaps study that holds on 3 indices?)